deps/rabbit/src/rabbit_vhost.erl

```diff
@@ -377,6 +410,10 @@ msg_store_dir_base() ->
     Dir = rabbit_mnesia:dir(),
     filename:join([Dir, "msg_stores", "vhosts"]).
 
+config_file_path(VHost) ->
+    VHostDir = msg_store_dir_path(VHost),
+    filename:join(VHostDir, ".config").
```

Comment:
We should avoid `.config` as it might generate confusion with `advanced.config` and such. How about `.meta` since all we store here is metadata?

Or did you intentionally want to emphasize that this is a `file:consult/1`-compatible file?

Reply:
I do want to show it's a consult file, but I agree that `.config` is not good, it's not something we want the user to ever modify. On that note I think I will add a comment saying it's auto generated inside the file.

Reply:
I have left `.config` and added a comment at the top of the file instead. I have noticed that we already have some `.config` for example https://github.com/rabbitmq/rabbitmq-server/blob/master/deps/rabbit/src/rabbit_node_monitor.erl#L73-L74

The `.config` for the queue is deep within the mnesia directory so it is unlikely that users stumble upon it. But if that's not good enough we can still change it, I'm not sure into what though.